R/db.explain.PrestoConnection.R

```diff
@@ -21,7 +21,7 @@ NULL
 #' }
 db_explain.PrestoConnection <- function(con, sql, ...) {
   build_sql <- dbplyr_compatible('build_sql')
-  explain.sql <- build_sql("EXPLAIN ", sql)
```

Comment:
Is this backwards compatible with dbplyr < 1.4.0?

Reply:
Yes, it should be. Just retested using 1.3.0 and everything passed.